### ISSUE TITLE
Sync problem

### ISSUE THREAD

**Issue by galaxydoctor** [lightninglabs/lightning-app#83]
### Description
I can't sync, and this message is in the log : "

lnd: 2017-10-26 14:48:32.715 [WRN] LTND: open ../lnd.conf: Le fichier spécifié est introuvable.
lnd: 2017-10-26 14:48:32.848 [INF] LTND: Version 0.3.0-alpha
lnd: 2017-10-26 14:48:32.854 [INF] CHDB: Checking for schema update: latest_version=0, db_version=0
lnd: 2017-10-26 14:48:32.854 [INF] LTND: Primary chain is set to: bitcoin
lnd: unable to create chain control: unable to create neutrino: EOF
lnd Error: unable to create neutrino: EOF
lnd: 2017-10-26 14:48:32.715 [WRN] LTND: open ../lnd.conf: Le fichier spécifié est introuvable.
lnd: 2017-10-26 14:48:32.848 [INF] LTND: Version 0.3.0-alpha
lnd: 2017-10-26 14:48:32.854 [INF] CHDB: Checking for schema update: latest_version=0, db_version=0
lnd: 2017-10-26 14:48:32.854 [INF] LTND: Primary chain is set to: bitcoin
lnd: unable to create chain control: unable to create neutrino: EOF
lnd Error: unable to create neutrino: EOF"

In appdata log file, I can read this : 

" [2017-10-13 22:58:18:0573] [error] lnd: unable to create neutrino: EOF

[2017-10-13 23:00:55:0117] [error] lnd: unable to create neutrino: EOF

[2017-10-13 23:00:55:0167] [error] lnd: unable to create neutrino: EOF

[2017-10-13 23:02:46:0414] [error] lnd: unable to create neutrino: EOF

[2017-10-13 23:03:34:0437] [error] lnd: unable to create neutrino: EOF

[2017-10-13 23:09:34:0394] [error] lnd: unable to create neutrino: EOF

[2017-10-13 23:09:56:0848] [error] lnd: unable to create neutrino: EOF

[2017-10-13 23:29:25:0913] [error] lnd: unable to create neutrino: EOF

[2017-10-13 23:47:12:0256] [error] lnd: unable to create neutrino: EOF

[2017-10-13 23:47:34:0552] [error] lnd: unable to create neutrino: EOF

[2017-10-13 23:47:48:0271] [error] lnd: unable to create neutrino: EOF

[2017-10-13 23:48:50:0794] [error] lnd: unable to create neutrino: EOF

[2017-10-14 00:09:25:0123] [error] lnd: unable to create neutrino: EOF

[2017-10-14 00:09:56:0577] [error] lnd: unable to create neutrino: EOF

[2017-10-22 19:47:07:0912] [error] lnd: unable to create neutrino: EOF

[2017-10-26 14:46:35:0785] [error] lnd: unable to create neutrino: EOF

"

### Steps to reproduce the behavior

Just lauching lightning

If you want any other informations, I can give it to you. I just don't know exactly what you could want. Then ask me :)

**Comment by galaxydoctor** [comment]
![image](https://user-images.githubusercontent.com/32781656/32054066-7e425764-ba5d-11e7-9b71-e81fe2ad9541.png)

**Comment by thejourneydude** [comment]
I'm running a similar issue on my Mac 10.12.6. Here are the following scenarios that I tested.

### Sync was successful
- Downloaded 0.1.1 Alpha version from Github release and opened up Electron app
- Ran `npm run package-electron` and opened up Electron app at current commit `daf039351b3d7a846c71381547e46d5385aa20c7`.
- Ran `npm run package-electron` and opened up Electron app at commit `e8368c09ec48050856cc2ac715f4174402b53814` which represents the 0.1.1 Alpha release commit.

### Sync never finishes
- Ran `npm start` and opened up Electron app at current commit `daf039351b3d7a846c71381547e46d5385aa20c7`.
- Ran `npm start` and opened up Electron app at commit `e8368c09ec48050856cc2ac715f4174402b53814` which represents the 0.1.1 Alpha release commit.

I also deleted the `Library/Application Support/Lnd` folder (I assume this is the data directory) and the sync still stalled.

Finally, here are the logs for both an app that synced and an app that didn't:

### Logs for Electron App that don't sync...
```
lnd: 2017-10-26 14:30:34.929 [WRN] LTND: open ../lnd.conf: no such file or directory 2017-10-26 14:30:34.930 [INF] LTND: Version 0.3.0-alpha 2017-10-26 14:30:34.938 [INF] CHDB: Checking for schema update: latest_version=0, db_version=0 2017-10-26 14:30:34.938 [INF] LTND: Primary chain is set to: bitcoin 2017-10-26 14:30:34.949 [INF] BTCN: Loaded 0 addresses from file '/Users/jason/Library/Application Support/Lnd/data/simnet/bitcoin/peers.json'
lnd: 2017-10-26 14:30:36.968 [INF] LNWL: Opened wallet
lnd: 2017-10-26 14:30:38.941 [INF] LNWL: The wallet has been unlocked without a time limit
lnd: 2017-10-26 14:30:38.943 [INF] LTND: LightningWallet opened
lnd: 2017-10-26 14:30:38.947 [INF] LNWL: Started rescan from block 683e86bd5c6d110d91b94b97137ba6bfe02dbbdb8e3dff722a669b5d69d77af6 (height 0) for 1 address 2017-10-26 14:30:38.947 [INF] LNWL: Catching up block hashes to height 0, this might take a while
lnd: 2017-10-26 14:30:38.947 [INF] LNWL: Done catching up block hashes 2017-10-26 14:30:38.947 [INF] LNWL: Finished rescan for 1 address (synced to block 683e86bd5c6d110d91b94b97137ba6bfe02dbbdb8e3dff722a669b5d69d77af6, height 0)
lnd: 2017-10-26 14:30:38.955 [INF] HSWC: Starting HTLC Switch 2017-10-26 14:30:38.955 [INF] RPCS: RPC server listening on 127.0.0.1:10009
lnd: 2017-10-26 14:30:38.956 [INF] NTFN: New block epoch subscription 2017-10-26 14:30:38.956 [INF] DISC: Authenticated Gossiper is starting 2017-10-26 14:30:38.956 [INF] CRTR: FilteredChainView starting 2017-10-26 14:30:38.956 [INF] CRTR: Prune tip for Channel Graph: height=0, hash=<nil> 2017-10-26 14:30:38.956 [INF] CRTR: Filtering chain using 0 channels active 2017-10-26 14:30:38.956 [INF] NTFN: New block epoch subscription 2017-10-26 14:30:38.956 [ERR] DISC: unable to rebroadcast stale channels: error while retrieving outgoing channels: no graph edges exist 2017-10-26 14:30:38.956 [INF] SRVR: Auto peer bootstrapping is disabled 2017-10-26 14:30:38.956 [INF] ATPL: Instantiating autopilot with cfg: (*main.autoPilotConfig)(0xc4204c9780)({ Active: (bool) true, MaxChannels: (int) 5, Allocation: (float64) 0.6 }) 2017-10-26 14:30:38.956 [INF] CMGR: Server listening on [::]:9735
lnd: 2017-10-26 14:30:38.957 [INF] ATPL: Autopilot Agent starting 2017-10-26 14:30:38.957 [INF] ATPL: Processing new external signal
lnd: 2017-10-26 14:30:38.957 [INF] RPCS: gRPC proxy started at localhost:8080
lnd: 2017-10-26 14:30:56.473 [INF] RPCS: [listchannels] fetched 0 channels from DB
lnd: 2017-10-26 14:31:16.528 [INF] RPCS: [listchannels] fetched 0 channels from DB
```

### Logs for Electron App that syncs
```
lnd: 2017-10-26 14:29:12.380 [WRN] LTND: open ../lnd.conf: no such file or directory 2017-10-26 14:29:12.380 [INF] LTND: Version 0.3.0-alpha
lnd: 2017-10-26 14:29:12.382 [INF] CHDB: Checking for schema update: latest_version=0, db_version=0 2017-10-26 14:29:12.382 [INF] LTND: Primary chain is set to: bitcoin
lnd: 2017-10-26 14:29:12.488 [INF] BTCN: Loaded 1004 addresses from file '/Users/jason/Library/Application Support/Lnd/data/testnet3/bitcoin/peers.json'
lnd: 2017-10-26 14:29:12.490 [INF] CMGR: DNS discovery failed on seed x19.seed.tbtc.petertodd.org: lookup x19.seed.tbtc.petertodd.org: no such host 2017-10-26 14:29:12.490 [INF] CMGR: DNS discovery failed on seed x19.testnet-seed.bitcoin.jonasschnelli.ch: lookup x19.testnet-seed.bitcoin.jonasschnelli.ch: no such host
lnd: 2017-10-26 14:29:12.532 [INF] CMGR: 3 addresses found from DNS seed testnet-seed.bluematt.me
lnd: 2017-10-26 14:29:12.685 [INF] BTCN: New valid peer 165.227.7.29:18333 (outbound) (/btcwire:0.5.0/btcd:0.12.0/)
lnd: 2017-10-26 14:29:12.699 [INF] BTCN: Syncing to block height 1211106 from peer 165.227.7.29:18333
lnd: 2017-10-26 14:29:13.518 [INF] LNWL: Opened wallet
lnd: 2017-10-26 14:29:12.685 [INF] BTCN: New valid peer 165.227.7.29:18333 (outbound) (/btcwire:0.5.0/btcd:0.12.0/)
lnd: 2017-10-26 14:29:12.699 [INF] BTCN: Syncing to block height 1211106 from peer 165.227.7.29:18333
lnd: 2017-10-26 14:29:13.518 [INF] LNWL: Opened wallet
lnd: 2017-10-26 14:29:14.437 [INF] LNWL: The wallet has been unlocked without a time limit
lnd: 2017-10-26 14:29:14.439 [INF] LTND: LightningWallet opened
lnd: 2017-10-26 14:29:14.440 [INF] LNWL: Started rescan from block 00000000be22dd05ef81913c2b4ab8cda7fb437ee93ca75709173df9489371a3 (height 1211106) for 1 address
lnd: 2017-10-26 14:29:14.441 [INF] LNWL: Catching up block hashes to height 1211106, this might take a while
lnd: 2017-10-26 14:29:14.441 [INF] LNWL: Done catching up block hashes 2017-10-26 14:29:14.441 [INF] LNWL: Finished rescan for 1 address (synced to block 00000000be22dd05ef81913c2b4ab8cda7fb437ee93ca75709173df9489371a3, height 1211106)
lnd: 2017-10-26 14:29:14.444 [INF] RPCS: RPC server listening on 127.0.0.1:10009
lnd: 2017-10-26 14:29:14.445 [INF] LTND: Waiting for chain backend to finish sync, start_height=1211106
lnd: 2017-10-26 14:29:14.445 [INF] LTND: Chain backend is fully synced (end_height=1211106)!
lnd: 2017-10-26 14:29:14.445 [INF] RPCS: gRPC proxy started at localhost:8080 2017-10-26 14:29:14.445 [INF] HSWC: Starting HTLC Switch
lnd: 2017-10-26 14:29:14.446 [INF] NTFN: New block epoch subscription
lnd: 2017-10-26 14:29:14.446 [INF] DISC: Authenticated Gossiper is starting 2017-10-26 14:29:14.446 [INF] NTFN: New block epoch subscription
lnd: 2017-10-26 14:29:14.446 [INF] CRTR: FilteredChainView starting
lnd: 2017-10-26 14:29:14.446 [INF] CRTR: Prune tip for Channel Graph: height=1211106, hash=00000000be22dd05ef81913c2b4ab8cda7fb437ee93ca75709173df9489371a3
lnd: 2017-10-26 14:29:14.447 [INF] CRTR: Filtering chain using 0 channels active
lnd: 2017-10-26 14:29:14.448 [INF] SRVR: Initializing peer network boostrappers!
lnd: 2017-10-26 14:29:14.448 [INF] CMGR: Server listening on [::]:9735
lnd: 2017-10-26 14:29:14.448 [INF] SRVR: Creating DNS peer boostrapper with seeds: [nodes.lightning.directory]
lnd: 2017-10-26 14:29:14.448 [INF] DISC: Attempting to bootstrap with: Authenticated Channel Graph
lnd: 2017-10-26 14:29:14.449 [INF] ATPL: Instantiating autopilot with cfg: (*main.autoPilotConfig)(0xc420160580)({ Active: (bool) true, MaxChannels: (int) 5, Allocation: (float64) 0.6 })
lnd: 2017-10-26 14:29:14.449 [INF] ATPL: Autopilot Agent starting
```

Obviously, I'm experiencing slightly different issues from galaxydoctor but I think it may be relevant.

**Comment by aakselrod** [comment]
I believe the commit at https://github.com/lightninglabs/neutrino/commit/dd2a98fb21bd13f70ec9e64a4bfec42093a732ba should fix the issue when it's included in the next release of the app.

**Comment by bryanvu** [comment]
@thejourneydude, I think you're issue is different than @galaxydoctor's, but it's also a good one. When you run the app with "npm start" it runs in dev mode on simnet, and when you package the app with "npm run package-electron", it runs in prod mode on testnet. When I switched the app from full node mode to neutrino mode in commit 5f2d3, i removed btcd. However, I now realize that in order to run in dev/simnet mode, we actually need that btcd to be added back in.

**Comment by thejourneydude** [comment]
Thanks for the clarification @bryanvu. I'll move this to a separate issue.

**Comment by galaxydoctor** [comment]
Hey @bryanvu  ! I've tried with the latest version (0.1.1) and there is the same result ... :/
I will reinstall my OS soon. I suspect something happened after the first time I've launched Lightning, because it worked one time. Only one time. The second time, this happened...

**Comment by galaxydoctor** [comment]
And I have deleted data ... :)

**Comment by aakselrod** [comment]
@galaxydoctor The first version of the app that will include the fix mentioned above will not be released until later today or possibly tomorrow. Version 0.1.1 does not include the fix.

**Comment by galaxydoctor** [comment]
@aakselrod Thank you ! 👍

**Comment by Roasbeef** [comment]
Should be fixed as of `0.1.3`, closing for now.
